### PR TITLE
CC-6358 Updates for the org move.

### PR DESCRIPTION
**Pull request by qubyte**
@lcapel & @antiBaconMachine Reopened with fix here.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +15,11 @@
...
+    "url": "https://github.com/vizia/vizia-transform-obj-to-series/issues"
   },
-  "homepage": "https://github.com/BrandwatchLtd/vizia-transform-obj-to-series#readme",
+  "homepage": "https://github.com/vizia/vizia-transform-obj-to-series#readme",
   "dependencies": {
-    "object-values": "^1.0.0"
+    "object-values": "1.0.0"
   },
   "devDependencies": {
     "standard": "^6.0.4",
```
Comment (lcapel) [inline package.json]: caret

Comment (qubyte) [inline package.json]: Won't fix.

**Comment by qubyte** [comment]
@lcapel Please ignore carets. They're outside the scope of this PR anyway.

**Comment by lcapel** [comment]
👍

**Comment by chrisnewtn** [comment]
👍

**Comment by chrisnewtn** [comment]
Hmm is it a problem that the test runner hasn't run for this repo, or is that out of scope?

**Comment by qubyte** [comment]
I'll try closing and opening to see if that triggers it.

**Comment by qubyte** [comment]
There we go. Just had to toggle it off and on from travis.

**Comment by qubyte** [comment]
@chrisnewtn build passed.

**Comment by chrisnewtn** [comment]
👍

**Pull request merged**